src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Github integration
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+
+<img
+  title="Github Integration dashboard"
+  alt="Github Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our Github integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+
+## Set up the Github integration [#set-up-github]
+
+Complete the following steps to set up the Github integration:
+
+<Steps>
+<Step>
+
+## Integration Details [#integration-details]
+
+By default, the Power Service catalog is selected, which will ensure to import all the data from Teams, Repositories and Users.
+    </Step>
+
+    <Step>
+
+## Github Integration [#github-integration]
+
+1. In this page, once we click on the install button, we get redirected to the Github App installation page. From there we need to install the Github app into the organization.
+2. If we are successfully able to install, it will show a success message with the teams, repo and users lists or else a failure message.
```

Comment:
A step is an action that a user performs. This is more like the possible outcome of a user action and therefore shouldn't be numbered.

Reply:
What happens after the setup? What's next for the user? Don't we need to show what kind of data is ingested in New Relic and how to make sense of that data? Take a look at this page: https://deploy-preview-19113--docs-website-netlify.netlify.app/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration/, for example. It lists all the metrics, events, and logs that one sees in New Relic as a result of the integration. Without that, this information would be incomplete.

Reply:
Added result page content!